### ISSUE TITLE
Blue/black usernames hardly readable in chat

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#588]
The blue/black for player one is hardly readable in the chat. At least I'm not able to read the username, since the background is also very dark. Either we have an opaque layer in the background, choose another blue/black or don't colour the usernames.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2019-04-30T09:57:08Z
 * owner: stonerl
 * the launchpad url was https://bugs.launchpad.net/bugs/1827028
